include/ignition/math/Plane.hh

```diff
@@ -119,6 +122,76 @@ namespace ignition
         return this->normal.Dot(_point) - this->d;
       }
 
+      /// \brief Given x,y point find corresponding Z point on plane.
+      /// \param[in] x - 2d x coordinate.
+      /// \param[in] y - 2d y coordinate.
+      /// \param[in] eps - The value to consider zero. Defaults to 1e-16
+      /// \return coincident point.
+      public: Vector3<T> GetPointOnPlane(const T x,
```

Comment:
* Style

    ```suggestion
      public: Vector3<T> PointOnPlane(const T x,
    ```

* Just reading the documentation, it took me a while to understand what the function was doing. Maybe saying something along the lines of "given a point on the horizontal XY plane (i.e. Z == 0), find that point projected on the plane"
* Can we use `Vector2` instead of separate x/y?
* Reminder to update style:
    * prefix inputs with `_`
    * use `camelCase`

Reply:
Removed in 45920e1 as this is not used anywhere and does not bring any happiness. Its probably from an older algorithm I was trying.